lib/jellyfish/config_reader.ex

```diff
@@ -180,6 +180,16 @@ defmodule Jellyfish.ConfigReader do
     end
   end
 
+  def read_git_commit() do
+    System.get_env("JF_GIT_COMMIT") ||
+      with {_path, 0} <- System.cmd("which", ["git"]),
+           {commit, 0} <- System.cmd("git", ["rev-parse", "HEAD"]) do
+        commit |> String.trim()
+      else
+        _error -> nil
```

Comment:
I think we should provide a default value here, e.g., "dev," or raise an exception; nil doesn't sound like a good value.